### DISCUSSION TITLE
Changed URL - Master Password no longer working [SOLVED]

### DISCUSSION THREAD

**Discussion by madtempest** [dani-garcia/vaultwarden#3229] [Q&A] [answered]
Hi All,

I'm migrating my Vaultwarden docker instance from one host to another and whilst a direct copy and paste of the docker container didnt work (not sure why, doesn't matter) I decided to just export and migrate instead.

As part of this, I named the new host with the url vaultwarden.domain.tld and I reconfigured the old host using the vaultwarden admin panel to vaultwarden-old.domain.tld.

Updated DNS etc and all seems good. I went to login to vaultwarden-old.domain.tld with my master password and it's failing to authenticate. Thankfully, I did have the browser plugin authenticated and I could export my vault from there, and when that prompted me for my master password for the export, the same password as I used on -old worked. The browser plugin was last authenticated using vaultwarden.domain.tld prior to the update of DNS records and has not been re-authenticated since.

Is the master password somehow sync'd to the domain name? Using the admin panel by bypassing the reverse proxy that gives the instance TLS, I can see the account is still there with items in it.

Thank you.

**Comment by BlackDex** [comment]
The password is not linked to any domain name except the one of the users email address.
Further, what do you see if you login into the admin interface and check the users in the new instance? Are the users there?

**Reply by madtempest** [reply]
The new instance is fine - it's the old instance thats the issue?

I found that doing a straight copy and paste from one container to another was seeing the new instance fail to start.

I will take a backup of the new instance (which is working on the new URL but its empty for now) and see if the copy and paste action works and reply back.

**Reply by BlackDex** [reply]
I do my testing both on a local url/ip and via a reverse-tunnel with a valid domain name on the exact same data. So that should work just fine.

Either the data/users are not there on the instance, or something else is wrong. But i suggest to check the admin panel, see if the users exists which you are expecting to exists. And check the logs of both Vaultwarden and your reverse proxy if all is reaching each other.

**Comment by madtempest** [comment, marked as answer]
So it looks like when I changed the URL, it wanted to verify the account again (for some reason). That and I had forgotten to update the URL in the admin panel behind the reverse proxy.

I can now access my old vault again to export it ready for the new vault.

Thanks for the help.
